src/main/java/com/freetimers/spartacus/gamebox/action/InfluenceForExhaustGladAction.java

```diff
@@ -0,0 +1,30 @@
+package com.freetimers.spartacus.gamebox.action;
+
+import java.util.Objects;
+
+public class InfluenceForExhaustGladAction implements Action {
+
+    private static final InfluenceForExhaustGladAction INSTANCE = new InfluenceForExhaustGladAction();
```

Comment:
Add field to indicate how much influence to increase. Pass influence to constructor